osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -70,8 +70,8 @@ private void load(LadderInfo ladder, TextureStore textures)
                 new FillFlowContainer
                 {
                     AutoSizeAxes = Axes.Both,
-                    Anchor = Anchor.TopCentre,
-                    Origin = Anchor.TopCentre,
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
```

Comment:
These make the text look off:

![image](https://user-images.githubusercontent.com/20418176/76168683-3fa27980-6172-11ea-940c-d1513e2972bb.png)

A few `TopLeft`s lower down and some spacing adjustments seem to bring this to a visually good-looking state.

Reply:
Weird merge failure (seems like this was fixed on the map pool screen PR rather than the base one).